src/config.js

```diff
@@ -322,6 +326,11 @@ function describeApi (rootDir) {
         language: LANGUAGE_JS_LATEST,
         requiredApi: jsDescribeApi({ rootDir })
       };
+    } else if (existsSync(path.join(rootDir, 'api.py'))) {
```

Comment:
Here the assumption is that we have either JS or Python functions, right? Isn't there an easy way to detect and parse both runtimes?